python/erdos/__init__.py

```diff
@@ -18,6 +18,8 @@
 import erdos.utils
 
 _num_py_operators = 0
+node_addrs = dict()
+num_procs = dict()
```

Comment:
Remove this global variable since we can calculate it from the previous dict (see below).